src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -2542,5 +2542,93 @@ public void CallingPartialValidationSucceeded_FoHeaderWithValidationStateAsParti
             headers.Should().BeNull();
             fullValidationRequired.Should().BeFalse();
         }
+
+        /// <summary>
+        /// Issue 43 @ CT is at 0. 10 headers are presented. 10 blocks are downloaded.
+        /// CT advances to 5. Alternative chain with fork at 3 and tip at 12 is presented.
+        /// Block data for alternative chain is downloaded. CT changes to block 8 of the 2nd chain.
+        /// Make sure that UnconsumedBlocksDataBytes is equal to the sum of
+        /// serialized sizes of 9b-12b + 6a-10a (a- first chain, b- second chain).
+        /// </summary>
+        [Fact]
+        public void
```

Comment:
Didn't you say that you did adjust your visual studio setting?

Reply:
My mistake this time - must have hit enter ;-)